src/scenarios/blazorminapp/pre.py

```diff
@@ -16,15 +16,22 @@
                 bin_dir=const.BINDIR,
                 exename=EXENAME,
                 working_directory=sys.path[0])
-if not precommands.has_workload:
-    subprocess.run(["dotnet", "workload", "install", "wasm-tools", "--skip-manifest-update"])
+
+# This project uses native relinking
+
+# We want to use native relinking which requires the workload
+precommands.install_workload('wasm-tools')
 f = open(os.path.join(os.getcwd(), "app", "emptyblazorwasmtemplate.csproj"), 'r')
 outFileText = ""
 for line in f.readlines():
     if "<PropertyGroup>" in line:
         outFileText += line
         outFileText += "    <BlazorEnableTimeZoneSupport>false</BlazorEnableTimeZoneSupport>" + os.linesep
         outFileText += "    <InvariantGlobalization>true</InvariantGlobalization>" + os.linesep
+        # this will trigger native relinking, and fail the build if the workload is not available
+        outFileText += "    <WasmBuildNative>true</WasmBuildNative>" + os.linesep
```

Comment:
@eerhardt Just wanted you to double check that this is the right goo to put in the csproj so that we are doing the minapp scenario correctly.

Reply:
Why didn't we need these properties before?

Reply:
Background:

- we recently started building these blazor apps with a sdk that has the `wasm-tools` workload (based on artifacts from a runtime build) already installed.
- Before this PR, the pre/post scripts would install/uninstall the workload, but we don't want to do that

When publishing a blazor project:
1. with the workload installed, native-relinking is used
2. without the workload, no native-relinking

- Using `WasmNativeWorkload=false` can simulate a workload not being available, so we use that to get behavior (2).

- For blazorminapp project we want relinking, for which we don't need to set anything extra.
  - BUT if due to any problem the sdk doesn't have the workload installed, then `publish` will still build successfully, but now silently not using native relinking.
  - Setting `WasmBuildNative=true` causes the workload to be *required*, thus failing the build if a workload is not installed.
  - So, essentially, this is just to enforce the intention of relinking